tests/common/tests/kalman_fitting_telescope_test.hpp

```diff
@@ -0,0 +1,85 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// Project include(s).
+#include "kalman_fitting_test.hpp"
+
+// Detray include(s).
+#include "detray/detectors/create_telescope_detector.hpp"
+#include "detray/io/common/detector_reader.hpp"
+#include "detray/io/common/detector_writer.hpp"
+
+namespace traccc {
+
+/// Combinatorial Kalman Finding Test with Sparse tracks
+class KalmanFittingTelescopeTests : public KalmanFittingTests {
+
+    public:
+    /// Plane alignment direction (aligned to x-axis)
+    static const inline detray::detail::ray<transform3> traj{
+        {0, 0, 0}, 0, {1, 0, 0}, -1};
+
+    /// Position of planes (in mm unit)
+    static const inline std::vector<scalar> plane_positions = {
```

Comment:
I would probably put a few of these options directly into the telescope config down below

Reply:
Makes sense but let;s do this next time...